### PR TITLE
Feature/issue 3 thinh

### PR DESCRIPTION
**Pull request by tcthinhsqt**

### CODE REVIEW

File: backend/app/Models/Student.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class Student extends Model
+{
+    protected $fillable = [
+        'name',
+        'birthday'
+    ];
+//    public function admins() {
+//        return $this->belongsTo('App\Models\Admin');
+//    }
```
Comment (tantai0809) [inline backend/app/Models/Student.php:15]: @tcthinhsqt 
please remove if unuse

**Review by tantai0809 — CHANGES_REQUESTED** [review]
@tcthinhsqt 
I have some comments, please check !!!

File: backend/app/Http/Requests/SearchStudentRequest.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        ];
+    }
+    public function messages()
+    {
+        return [
+
+        ];
+    }
+}
```
Comment (tantai0809) [inline backend/app/Http/Requests/SearchStudentRequest.php:37]: @tcthinhsqt 
please remove if don't use

**Review by tantai0809 — CHANGES_REQUESTED** [review]
I have comment, but please fix it in next issue

File: backend/app/Http/Requests/SearchStudentRequest.php (hunk 2 above)
Comment (tcthinhsqt) [reply backend/app/Http/Requests/SearchStudentRequest.php:37]: ok i got it

**Pull request closed without merging**